compiler/fir/resolve/src/org/jetbrains/kotlin/fir/resolve/dfa/cfg/ControlFlowGraphBuilder.kt

```diff
@@ -793,7 +796,7 @@ class ControlFlowGraphBuilder {
 
         if (tryExpression.finallyBlock != null) {
             val finallyEnterNode = createFinallyBlockEnterNode(tryExpression)
-            addEdge(enterTryNodeBlock, finallyEnterNode)
+            addEdge(enterTryNodeBlock, finallyEnterNode, label = "${FINALLY_PREFIX}_${tryExitNode.level}")
```

Comment:
I just realize that we actually have only two different labels: 
- after finally go to function exit
- after finaly go to next line

So maybe it's ok to replace type of label with some enum or even add two new EdgeKinds? This way may help us to save some memory and don't create new entities just for one case, which simplify understating of this part of analisys

Reply:
`enum` sounds good to me. That also removed `K: Any?` change I added to control-flow info. Plus, APIs to check/retrieve the property init info at the normal path look much simpler too. Thanks for the suggestion!